src/Microsoft.Windows.Compatibility/src/Microsoft.Windows.Compatibility.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <!-- Using a csproj extension to get the correct multi-targeting behavior. -->
-    <TargetFrameworks>$(NetCurrent);$(NetPrevious);$(NetMinimum);netstandard2.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net9.0;net8.0;net6.0;netstandard2.1;netstandard2.0</TargetFrameworks>
```

Our packages (in dotnet/runtime and System.Drawing.Common and Compatiblity) don't support net6.0 anymore as .NET 6 and .NET 7 will be out-of-support when .NET 9 ships.

```suggestion
    <TargetFrameworks>net9.0;net8.0;netstandard2.1;netstandard2.0</TargetFrameworks>
```